modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -24,17 +24,22 @@ Use the following procedure to configure CD-ROMs for virtual machines.
 .Procedure
 
 . From the *{object-gui}* tab, select your {object}.
+
 . Select the *Overview* tab.
-. Click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
-* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
-* If there are CD-ROMs available, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+
+. To add or edit a CD-ROM configuration, click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
+
+* If CD-ROMs are unavailable for editing, the message displays: *The virtual machine doesn't have any CD-ROMs attached*.
```

Comment:
s/the message/the following message

Reply:
Is there punctuation after *attached* in the UI? If so, include it before the `*`

Reply:
Added punctuation.